tests/queries/0_stateless/02136_scalar_progress.reference

```diff
@@ -2,5 +2,5 @@
 < X-ClickHouse-Progress: {"read_rows":"65505","read_bytes":"524040","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"0","result_bytes":"0"}
 < X-ClickHouse-Progress: {"read_rows":"131010","read_bytes":"1048080","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"0","result_bytes":"0"}
 < X-ClickHouse-Progress: {"read_rows":"131011","read_bytes":"1048081","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"0","result_bytes":"0"}
-< X-ClickHouse-Progress: {"read_rows":"131011","read_bytes":"1048081","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"1","result_bytes":"80"}
-< X-ClickHouse-Summary: {"read_rows":"131011","read_bytes":"1048081","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"1","result_bytes":"80"}
+< X-ClickHouse-Progress: {"read_rows":"131011","read_bytes":"1048081","written_rows":"0","written_bytes":"0","total_rows_to_read":"100000","result_rows":"1","result_bytes":"272"}

```

Comment:
Why has this PR changed the return bytes? Why is it dependent on the padding?

Reply:
Looking into it it seems that ColumnNumber uses PaddedPODArray as storage and this grew in size due to the extra padding (used to have 64B of padding and now it has 256B), so I guess it's expected.

Not sure why this is marked as not for changelog when the behaviour changes (as we can see in the tests) and CH uses more memory. I'm not saying that's wrong, but a notice would be nice.

Reply:
I don't think extra 200B memory consumption per column is a significant change for users

Reply:
I agree that's not significant, but it's changing the behaviour of the database. In our case we detected it because of integration tests.

I think my expectations about what should be in the changelog are different to the project's guidelines, because I keep having to look back into PRs that made noticeable (and sometimes really important) changes and they never made it to the changelog.

Reply:
Well, this was added to the changelog anyway. So sorry for the confusion and the noise.